src/2_RTB/AnalisiDeiRequisiti/content.typ

```diff
@@ -47,7 +47,7 @@ La piattaforma prevede fondamentalmente una tipologia di utente: l'amministrator
 La Proponente consiglia l'utilizzo delle seguenti tecnologie per la realizzazione del progetto:
 - Implementazione dei simulatori. Tecnologia consigliata: *Python*;
 - Database OLAP utilizzato per lo storage dei dati.  Tecnologia consigliata: *ClickHouse*;
-- Piattaforma di stream processing. Tecnologia consigliata: *Kafka*.
+- Piattaforma di stream processing. Tecnologia consigliata: *Kafka*;
 - Dashboard di visualizzazione. Tecnologia consigliata: *Grafana*.
```

Comment:
Togli pure le tecnologie consigliate, ho guardato esempi sia di AdR che di PdP e non le ho trovate incluse in nessuno dei due; potremmo sempre scegliere di tenerle ma non ha tantissimo senso in questo contesto.